Competition2017/src/competition/subsystems/climbing/ClimbingSubsystem.java

```diff
@@ -56,6 +58,14 @@ public void descend() {
     public void setBrake(boolean brakeOn) {
         brakeSolenoid.set(brakeOn);
     }
+    
+    public boolean isLimitSwitchPressed() {
+        return climbingMotor.isForwardLimitSwitchClosed();
+    }
+    
+    public void setEnableSafeties(boolean isEnabled) {
+        climbingMotor.enableLimitSwitches(isEnabled, isEnabled);
```

Comment:
Earlier, the reverse limit switch is configured to be disabled universally; I think we want to do that here as well.